### ISSUE TITLE
[Bug]: WebRTC page auto-refreshing and not connecting on iPhone 12 Pro

### ISSUE THREAD

**Issue by jonzinho** [thinkonmay/thinkremote-v2#6]
### What happened?

When I try to use the WebRTC link provided on iPhone it doesn't work, but on Android and Windows it works perfectly.

### To reproduce the behavior?

Just try to connect using an iOS device.

### Which device / platform are you seeing the problem on?

IOS

### What browser are you seeing the problem on?

Safari

**Comment by pigeatgarlic** [comment]
Hmmm although this bug is very critical, I could not identify this bug yet, this could be due to your network condition

**Comment by pigeatgarlic** [comment]
one question, when you connnect, there are some text pop up lik exchanging signaling message right? at which stage does the page start to reload?

**Comment by jonzinho** [comment]
> one question, when you connnect, there are some text pop up lik exchanging signaling message right? at which stage does the page start to reload?

yes, it start to reload after the exchanging signaling i think, i need to test it again to be sure

**Comment by jonzinho** [comment]
in my android system and windows it works perfectly

**Comment by pigeatgarlic** [comment]
Normally, the process would be like: websocket connected -> exchanging signalling message -> webrtc connected  -> received video stream

**Comment by pigeatgarlic** [comment]
At which stage your connection is failing ?

**Comment by pigeatgarlic** [comment]
hi @jonzinho, I have tested the connection on iphone Xs and it is working fine, so I suspect this issue related to your network condition rather than OS issue or browser issue
